crates/core_arch/src/x86/test.rs

```diff
@@ -1,115 +1,228 @@
 //! Utilities used in testing the x86 intrinsics
 
+use crate::core_arch::assert_eq_const as assert_eq;
 use crate::core_arch::x86::*;
 use std::mem::transmute;
 
 #[track_caller]
 #[target_feature(enable = "sse2")]
-pub unsafe fn assert_eq_m128i(a: __m128i, b: __m128i) {
+#[rustc_const_unstable(feature = "stdarch_const_helpers", issue = "none")]
```

Comment:
Is this actually needed? These functions aren't exported and shouldn't need stability attributes.